ci/scripts/common.env.sh

```diff
@@ -1,3 +1,4 @@
 export CARGO_TERM_COLOR=always
 export RUSTFLAGS="-D warnings"
 export PROTOC_NO_VENDOR=true
+export RISINGWAVE_CI=true
```

Comment:
?

Reply:
The unit test need to use ext4/xfs fs. With CI environment, I reuse the `/risingwave` directory to create a tempdir.